src/main/scala/org/scalawag/jibe/Reporter.scala

```diff
@@ -131,5 +132,15 @@ object Reporter {
         </html>
       )
     }
+
+    try {
+      val symlinkPath = symlink.toPath
+      Files.deleteIfExists(symlinkPath)
+      Files.createSymbolicLink(symlinkPath, symlinkPath.getParent.relativize( output.toPath ))
+    } catch {
+      case uoe: UnsupportedOperationException => println("Your OS sucks. Got symlinks?")
+      case unknown: Throwable => println("Failed to Create symlink: " + unknown)
```

Comment:
Throwable is a little broad for this.  Usually, Exception is what you'd catch in the normal course of things.  Throwable includes things like OutOfMemoryError and things that can't be caught and handled reasonably.